.github/workflows/blackbox-testing.yml

```diff
@@ -12,28 +12,38 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
     steps:
-      - name: Checkout repository
+      - if: "github.event.pull_request.draft == false &&!((github.actor == 'asyncapi-bot' && startsWith(github.event.pull_request.title, 'ci: update global workflows')) || (github.actor == 'asyncapi-bot' && startsWith(github.event.pull_request.title, 'chore(release):')) || (github.actor == 'allcontributors' && startsWith(github.event.pull_request.title, 'docs: add')))"
```

Comment:
Is there something like global condition on job? I don't think so that we need to check  `steps.should_run.outputs.shouldRun == 'true'` on every step

Reply:
Good call 🤔 Let me check